packages/base/all/vendor-config-onl/src/python/onl/install/BaseInstall.py

```diff
@@ -656,6 +658,22 @@ def installGrubCfg(self):
         ctx['boot_menu_entry'] = sysconfig.installer.menu_name
         ctx['boot_loading_name'] = sysconfig.installer.os_name
 
+        if self.isUEFI:
+            ctx['set_root_para'] = "set root='(hd0,gpt1)'"
```

Comment:
This will likely need to be parameterized. But lets accept as is to get the initial work going. 
Thank you for initial UEFI support!